src/citrine/resources/material_run.py

```diff
@@ -19,7 +19,38 @@
 
 
 class MaterialRun(DataConcepts, Resource['MaterialRun'], TaurusMaterialRun):
-    """A material run."""
+    """
+    A material run.
+
+    Parameters
+    ----------
+    name: str
+        Name of the material run.
+    uids: Map[str, str], optional
+        A collection of unique identifiers, each a key-value pair. The key is the "scope"
+        and the value is the identifier. The scope "id" is reserved for the internal Citrine ID,
+        which will always be a uuid4.
+    tags: List[str], optional
+        A set of tags. Tags can be used for filtering.
+    notes: str, optional
+        Long-form notes about the material run.
+    process: ProcessSpec
+        Process that produces this material
+    sample_type: str, optional
+        The form of this sample. Optionals are "experimental", "virtual", "production", or
+        "unknown." Default is "unknown."
```

Comment:
```suggestion
        "unknown". Default is "unknown".
```
Grammatically incorrect, I think, but more clear from a technical perspective.

Reply:
Dare we violate the rules of grammar? I agree it is more clear. If we change it here we should also make the equivalent change in the three attribute classes, the `origin` field.